README.md

```diff
@@ -300,7 +307,7 @@ Given example
 
 fastify.register(FastifyReplyFrom, {
   base: 'http://localhost:3001/',
-  customRetry: {handler: customRetryLogic, retries: 10}
+  customRetry: {handler: customRetryLogic}
```

Comment:
Can we avoid this options object taking only one property, and just accept the function directly? Sorry to be annoying!
```suggestion
  retryDelay: customRetryLogic
```